### PR TITLE
Update k8s-system-api-server.json

### PR DESCRIPTION
**Pull request by Nello-Angelo**
I added the ability to view the metrics for kubernetes clusters and server api nodes separately, as this adds more visibility

### CODE REVIEW

**Comment by Nello-Angelo** [comment]
i can do the same for other dashboards

File: dashboards/k8s-system-api-server.json [hunk 1]
```diff
@@ -1,41 +1,4 @@
 {
-  "__inputs": [
```
Comment (dotdc) [inline dashboards/k8s-system-api-server.json:2]: This is needed for Grafana.com

File: dashboards/k8s-system-api-server.json [hunk 2]
```diff
@@ -61,82 +24,96 @@
   "description": "This is a modern API Server dashboard for your Kubernetes cluster(s). Made for kube-prometheus-stack and take advantage of the latest Grafana features. GitHub repository: https://github.com/dotdc/grafana-dashboards-kubernetes",
   "editable": true,
   "fiscalYearStartMonth": 0,
+  "gnetId": 15761,
```
Comment (dotdc) [inline dashboards/k8s-system-api-server.json:27]: remove gneId and id

File: dashboards/k8s-system-api-server.json [hunk 3]
```diff
@@ -61,82 +24,96 @@
...
   "graphTooltip": 1,
+  "id": 15,
   "links": [],
   "liveNow": false,
   "panels": [
     {
       "datasource": {
         "type": "prometheus",
-        "uid": "${datasource}"
+        "uid": "P4169E866C3094E38"
```
Comment (dotdc) [inline dashboards/k8s-system-api-server.json:36]: This is your hardcoded version, should be ${datasource}

**Review by dotdc — CHANGES_REQUESTED** [review]
Thank you for submitting this PR, unfortunately, there are a few issues right now :
- your base version is old, and you revert a lot of changes
- you remove useful information like __inputs
- you add instance specific information from your setup (hardcoded uid, id, gneid...)
- there are several changes in this PR, and it's unclear what the goal is.

If the goal was to fix https://github.com/dotdc/grafana-dashboards-kubernetes/issues/15, I think you should read the issue again. The cluster variable is not universal, and I don't want to make it a prerequisite for this project. 

Like mentioned before, a script that could generate and inject that in the dashboards would be good for users who cannot rely on Grafana datasources to manage their clusters. 

I will explain the workaround in the documentation to make things more clear.

**Comment by Nello-Angelo** [comment]
okay i will clouse the PR

**Pull request closed without merging**